content/metadata.yaml

```diff
@@ -651,12 +651,12 @@ author_info:
 #   twitter: null
 #   affiliations:
 #   - Altius Institute
-# - name: Jason Miller
-#   orcid: null
-#   github: null
-#   twitter: null
-#   affiliations:
-#   - University of Pennsylvania
+ - name: Jason Miller
```

Comment:
If this build fails, please try removing a single space from the beginning of each of these lines you edited.